docs/source/resources/package-spec.rst

```diff
@@ -100,9 +99,9 @@ conda package is composed of the first 3 values, as in:
        the package.
 
        Unlike the build string, the ``build_number`` is inspected by
-       conda. Conda uses it to sort packages that have otherwise
-       identical names and versions to determine the latest one.
-       This is important because new builds that contain bug
+       conda. Conda uses |br| it to sort packages that have otherwise
+       identical names and versions |br| to determine the latest one.
+       This is important because new builds that |br| contain bug
```

Comment:
It's better to **not** hard-code line breaks like this, as it hard-codes rendering features into the source files. Common practice in rST files is to just add a line break which will be rendered as a line-break, e.g.:

```suggestion
       conda.
       
       Conda uses it to sort packages that have otherwise
       identical names and versions to determine the latest one.

       This is important because new builds that contain bug
```